lib/api/listMultipartUploads.js

```diff
@@ -81,17 +81,28 @@ const _convertToXml = xmlParams => {
     xml.push('<?xml version="1.0" encoding="UTF-8"?>',
              '<ListMultipartUploadsResult ' +
                 'xmlns="http://s3.amazonaws.com/doc/2006-03-01/">',
-             `<Bucket>${xmlParams.bucketName}</Bucket>`,
-             `<KeyMarker>${escapeForXML(xmlParams.keyMarker)}</KeyMarker>`,
-             `<UploadIdMarker>${xmlParams.uploadIdMarker}</UploadIdMarker>`,
-             `<NextKeyMarker>${escapeForXML(l.NextKeyMarker)}</NextKeyMarker>`,
-             `<NextUploadIdMarker>${l.NextUploadIdMarker}</NextUploadIdMarker>`,
-             `<Delimiter>${escapeForXML(l.Delimiter)}</Delimiter>`,
-             `<Prefix>${escapeForXML(xmlParams.prefix)}</Prefix>`,
-             `<MaxUploads>${l.MaxKeys}</MaxUploads>`,
-             `<IsTruncated>${l.IsTruncated}</IsTruncated>`
+             `<Bucket>${xmlParams.bucketName}</Bucket>`
     );
 
+    const params = [
+        { tag: 'KeyMarker', value: escapeForXML(xmlParams.keyMarker) },
+        { tag: 'UploadIdMarker', value: xmlParams.uploadIdMarker },
+        { tag: 'NextKeyMarker', value: escapeForXML(l.NextKeyMarker) },
```

Comment:
Should we be returning any xml tags for this or just nothing?  It appears on a regular listing we should not return NextMarker so I wonder if this is the same.